docs/stencil-docs/storefront-customization/WCAG-compliance-levels.md

```diff
@@ -0,0 +1,82 @@
+# WCAG Compliance Levels
+<div class="otp" id="no-index">
+
+### On this page
+- [Accessibility principles](#accessibility-principles)
+- [Accessibility guidelines](#accessibility-guidelines)
+- [Related resources](#related-resources)
+</div>
+
+This article demonstrates how Cornerstone satisfies [Web Content Accessibility Guidelines (WCAG) 2.1](https://www.w3.org/TR/WCAG21/#guidelines) across compliance levels. You can perform similar techniques to your site regardless of your compliance level. For detailed implementation information, see [Implementing WCAG Guidelines]().
+
+## Accessibility principles
+Accessibility content must be POUR as stated by WCAG which organizes accessibility guidelines into four principles.
+
+P = Perceivable
+
+O = Operable
+
+U = Understandable
+
+R = Robust
+
+**Perceivable** content means users can comprehend the information being presented. 
+
+*The information cannot be invisible to all their senses.*
+
+**Operable** content means users can perform actions using interface components and navigate the site.
+
+*The interface cannot require interaction that a user cannot perform.*
+
+**Understandable** content means the information and operation of a user interface is understandable.
+
+*The operation of interface or information cannot cause confusion or discomfort for users.*
+
+**Robust** content means the content can be used by a wide variety of technologies.
+
+*The content cannot cause content inaccessibility.*
+
+You are encouraged to create content that addresses all four principles. If any principle is missing, users with disabilities will not be able to successfully use and access web content. 
+
+## Accessibility guidelines
+Each principle contains guidelines. The guidelines are the goals you should work towards to acheive a certain conformance level. 
+The guidelines featured below are not a complete list and were chosen to highlight possible solutions for how to meet requirements. Using Cornerstone as an example, this article demonstrates how Cornerstone meets the success criteria for the following guidelines.
+ 
```

Comment:
For all the entries in the "Cornerstone technique," Avoid using the word "Cornerstone." Instead, go for phrases that still provide clear description of the technique/feature.